DistributionPackages/Neos.NeosIo/Resources/Private/Templates/Page/Default.html

```diff
@@ -37,26 +57,27 @@
 </f:section>
 <f:section name="bodyScripts">
 	<script src="{f:uri.resource(path: 'JavaScript/Bundle.min.js', package: 'Neos.NeosIo')}" async></script>
-	<f:if condition="{googleAnalyticsTrackingCode}">
-		<script>
-			<![CDATA[
-			(function(i, s, o, g, r, a, m) {
-				i['GoogleAnalyticsObject'] = r;
-				i[r] = i[r] || function() {
-						(i[r].q = i[r].q || []).push(arguments)
-					}, i[r].l = 1 * new Date();
-				a = s.createElement(o),
-					m = s.getElementsByTagName(o)[0];
-				a.async = 1;
-				a.src = g;
-				m.parentNode.insertBefore(a, m)
-			})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
-			]]>
-
-			ga('create', '{googleAnalyticsTrackingCode}', 'auto');
-			ga('send', 'pageview');
+	<f:if condition="{neos:rendering.live()}">
```

Comment:
You don't need the tracking code anymore if you have the TagManager, as you configure the tracking there.